### PR TITLE
[absent] Add new port

### PR DESCRIPTION
**Pull request by rvarago**
**Describe the pull request**
This PR adds a new port for the library: absent. absent is a small library meant to simplify the composition of nullable types in a generic, type-safe, and declarative style
for some C++ type-constructors

- What does your PR fix? Fixes #
N/A

- Which triplets are supported/not supported? Have you updated the CI baseline?
I believe it should work fine for all triplets as long as the compiler supports C++17 or greater.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
I hope it does. Please let me know if otherwise.

### CODE REVIEW

File: ports/absent/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,35 @@
+include(vcpkg_common_functions)
```
Comment (LilyWangL) [inline ports/absent/portfile.cmake:1]: `include(vcpkg_common_functions)` is deprecated. Please remove it.

File: ports/absent/CONTROL [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Source: absent
+Version: 0.3.0
```
Comment (LilyWangL) [inline ports/absent/CONTROL:2]: Can you please add the Homepage for this port in the CONTROL file?

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/absent/portfile.cmake (hunk 1 above)
Comment (rvarago) [reply ports/absent/portfile.cmake:1]: Sure, I am going to remove it. Thanks.
Done.

File: ports/absent/CONTROL (hunk 2 above)
Comment (rvarago) [reply ports/absent/CONTROL:2]: Good catch! I am going to add it. Thanks.
Done.

**Review by LilyWangL — APPROVED** [review]

**Comment by strega-nil** [comment]
Perfect, thanks much @rvarago :D

**Pull request merged**